openwisp_controller/connection/tests/base.py

```diff
@@ -51,6 +51,14 @@ def _create_device(self, *args, **kwargs):
             })
         return super()._create_device(*args, **kwargs)
 
+    def _get_credentials(self, **kwargs):
```

Comment:
in other modules we have similar methods but they're called `_get_or_create`, can you name it like that please?

Reply:
I have named it `_get_credentials` because I found a lot of `_get_xxxx`. Do you want me to rename them too?

Reply:
Thanks for bringing it up. I found those methods in openwisp-users.

Keeping the entire codebase consistent requires some effort, but things like this are not urgent it now. If we find out many inconsistencies we can write Google Code-In tasks for improving consistency.

Let's leave it.